pipelines/performance.yml

```diff
@@ -90,47 +88,23 @@ templates:
-      RM_CASE_SERVICE_PORT: '80'
-      RM_COLLECTION_EXERCISE_SERVICE_HOST: rm-collection-exercise-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RM_COLLECTION_EXERCISE_SERVICE_PORT: '80'
-      RM_IAC_SERVICE_HOST: iac-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RM_IAC_SERVICE_PORT: '80'
-      RM_SAMPLE_SERVICE_HOST: rm-sample-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RM_SAMPLE_SERVICE_PORT: '80'
-      RM_SURVEY_SERVICE_HOST: rm-survey-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RM_SURVEY_SERVICE_PORT: '80'
-      RAS_CASE_SERVICE_HOST: rm-case-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_CASE_SERVICE_PORT: '80'
-      RAS_COLLEX_SERVICE_HOST: rm-collection-exercise-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_COLLEX_SERVICE_PORT: '80'
-      RAS_IAC_SERVICE_HOST: iac-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_IAC_SERVICE_PORT: '80'
-      RAS_OAUTH_SERVICE_HOST: ras-rm-auth-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_OAUTH_SERVICE_PORT: '80'
-      RAS_PARTY_SERVICE_HOST: ras-party-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_PARTY_SERVICE_PORT: '80'
-      RAS_COLLECTION_INSTRUMENT_SERVICE_HOST: ras-collection-instrument-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_COLLECTION_INSTRUMENT_SERVICE_PORT: '80'
-      RAS_PUBLIC_WEBSITE_HOST: ras-frontstage-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_PUBLIC_WEBSITE_PORT: '80'
-      RAS_SECURE_MESSAGE_SERVICE_HOST: ras-secure-message-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_SECURE_MESSAGE_SERVICE_PORT: '80'
-      RAS_SECURE_MESSAGING_SERVICE_HOST: ras-secure-message-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
-      RAS_SECURE_MESSAGING_SERVICE_PORT: '80'
+      SAMPLE_SERVICE_HOST: rm-sample-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      SAMPLE_SERVICE_PORT: '80'
+      OAUTH_SERVICE_HOST: ras-rm-auth-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      OAUTH_SERVICE_PORT: '80'
+      FRONTSTAGE_HOST: ras-frontstage-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      FRONTSTAGE_PORT: '80'
+      SECURE_MESSAGING_SERVICE_HOST: ras-secure-message-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      SECURE_MESSAGING_SERVICE_PORT: '80'
       SECURE_MESSAGE_URL: http://ras-secure-message-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
```

Comment:
This one is inconsistent with the 2 above

Reply:
There are 2 there's secure message and secure messaging I left them both as I didn't know which one is needed